blog/index.md

```diff
@@ -3,6 +3,17 @@ title: Blog
 layout: blog-index
 ---
 
+# [Deploy Skydive on top of OpenStack thanks to Tripleo](/blog/tripleo.html)
+## by Sylvain Afchain, 07/08/2018
+
+Skydive supports multiple deployment ways, from containers(Kubernetes, OpenShift) to Ansible playbook. In this blogpost I will explain how to
+deploy Skydive on top of OpenStack using Tripleo. Skydive is already integrated as part of the Tripleo deployment since the Queens realase but
+has been reworked in order to leverage the new 
+[config download](https://docs.openstack.org/tripleo-docs/latest/install/advanced_deployment/ansible_config_download.html).
+During the last few weeks we added a Tripleo job in our CI, in this blogpost I will extract some of the scripts involved in
```

Comment:
During the last few weekds, we added a TripleO job to our CI. In this blog post, I will extract some of the scripts involved in the CI job to show how to deploy the latest version of Skydive with Tripleo.